### PR TITLE
ICMSLST=955 Use one sidebar to create/edit/submit applications

### PR DESCRIPTION
**Pull request by kevinetienne**

### CODE REVIEW

File: web/templates/web/domains/case/export/cfs-edit-schedule.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {% from "display/fields.html" import application_section, application_field %}
 
 {% block sidebar %}
-  {% include "partial/case/export/cfs/sidebar.html" %}
+  {% include "partial/case/sidebar.html" %}
```
Comment (kevinetienne) [inline web/templates/web/domains/case/export/cfs-edit-schedule.html:6]: delete please

File: web/templates/partial/case/link-update-requests.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{% if process.status == "UPDATE_REQUESTED" %}
+  <li>
+    <a
+      href="{{ icms_url('case:list-update-requests', kwargs={'application_pk': process.pk, 'case_type': 'import'}) }}">
```
Comment (MattHolmes123) [inline web/templates/partial/case/link-update-requests.html]: Should this be `'case_type': case_type`

**Review by MattHolmes123 — CHANGES_REQUESTED** [review]

File: web/templates/partial/case/link-update-requests.html (hunk 2 above)
Comment (kevinetienne) [reply web/templates/partial/case/link-update-requests.html]: Yes it should be but I've removed this file from this PR as it should be coming with the update request code.

Comment (MattHolmes123) [reply web/templates/partial/case/link-update-requests.html]: No worries.
I thought it looked like new code :)

**Review by MattHolmes123 — APPROVED** [review]

**Review by mikeylockwood — APPROVED** [review]

**Pull request merged**